### ISSUE TITLE
Refresh method doesn't return the full dataset

### ISSUE THREAD

**Issue by roblabla** [zekesonxx/node-yggdrasil#2]
It'd be nice if refresh could return the full dataset as a third argument to the callback, e.g.

```js
  r.refresh = function (access, client, cb) {
    r._call('refresh', {
      accessToken: access,
      clientToken: client
    }, function (err, data) {
      if (data.clientToken !== client) {
        cb("clientToken assertion failed");
      } else {
        cb(null, data.accessToken, data);
      }
    });
  };
```

I need access to the selectedProfile hash in node-minecraft-protocol, which your library doesn't provide.

**Comment by zekesonxx** [comment]
If you add that in, add the documentation for it, and PR it I'll be happy to merge it

**Comment by roblabla** [comment]
#3 and #4 are up :D

**Comment by zekesonxx** [comment]
Closed by #4
